compiler/ballerina-parser/src/main/java/io/ballerina/compiler/internal/parser/BallerinaParserErrorHandler.java

```diff
@@ -750,6 +750,9 @@
                     ParserRuleContext.QUALIFIED_IDENTIFIER_PREDECLARED_PREFIX };
 
     private static final ParserRuleContext[] MODULE_VAR_DECL_START =
+            { ParserRuleContext.VAR_DECL_STMT, ParserRuleContext.PUBLIC_QUAL, ParserRuleContext.MODULE_VAR_FIRST_QUAL };
+
+    private static final ParserRuleContext[] MODULE_VAR_DECL_PUBLIC_QUAL_RHS =
```

Comment:
Do we need this change? `MODULE_VAR_DECL_START` is the next rule of `MODULE_VAR_DECL` and `MODULE_VAR_DECL` is called by `TOP_LEVEL_NODE_WITHOUT_MODIFIER`. Without modifier means we have to exclude `public`.

Reply:
Yes, we don't need it